logstash-core/lib/logstash/runner.rb

```diff
@@ -79,6 +79,15 @@ class LogStash::Runner < Clamp::Command
     I18n.t("logstash.runner.flag.node_name"),
     :attribute_name => :node_name
 
+  def validate_positive_integer(str_arg)
+    int_arg = str_arg.to_i
+    if str_arg !~ /^\d+$/ || int_arg < 1
```

Comment:
oh man, i don't know what i was thinking there (facepalm) I swear i read it as `int_arg < 0`